sofdev6-02-2024/LosAderezos
# String Location

To test:
docker compose up --build
Then you go to the Docker Desktop, to the ProductManagementDB container
When you enter this container go to the column that says Exec and execute the following commands in order;
1. mysql -u root -p 
2. admin
3. use dbaderezosdev;
4. show tables;
5. select * from Subsidiary;

Then you can see all the subsidiaries with the changed attribute.
